modules/Students/applicationForm_manage_edit_fee.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/*
+Gibbon, Flexible & Open School System
+Copyright (C) 2010, Ross Parker
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+use Gibbon\Forms\Form;
+use Gibbon\Services\Format;
+use Gibbon\Domain\Students\ApplicationFormGateway;
+
+// Module includes
+require_once __DIR__ . '/moduleFunctions.php';
+
+if (isActionAccessible($guid, $connection2, '/modules/Students/applicationForm_manage_edit.php') == false) {
+    // Access denied
+    echo Format::alert(__('You do not have access to this action.'));
+} else {
+    // Proceed!
+    $gibbonApplicationFormID = $_GET['gibbonApplicationFormID'] ?? '';
+    $gibbonSchoolYearID = $_GET['gibbonSchoolYearID'] ?? '';
+    $search = $_GET['search'] ?? '';
+
+    $urlParams = compact('gibbonApplicationFormID', 'gibbonSchoolYearID', 'search');
+
+    $page->breadcrumbs
+        ->add(__('Manage Applications'), 'applicationForm_manage.php', $urlParams)
+        ->add(__('Edit Form'), 'applicationForm_manage_edit.php', $urlParams)
+        ->add(__('Send Payment Request'));
+
+    if ($gibbonApplicationFormID == '' or $gibbonSchoolYearID == '') {
+        echo Format::alert(__('You have not specified one or more required parameters.'));
+        return;
+    }
+
+    $application = $container->get(ApplicationFormGateway::class)->getByID($gibbonApplicationFormID);
+    if (empty($application)) {
+        echo Format::alert(__('The specified record does not exist.'));
+        return;
+    }
+
```

Comment:
Is it worth having a check to see if payment2 has been made yet, in which case the screen should not be available?

Reply:
Yep, good call, I'll add the check here too.